content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -0,0 +1,331 @@
+
+This is an article about Sockets. It will give you a glimpse of how clients and servers communicate over the internet.
+
+Table of content:
+- [What is a socket](#what-is-a-socket)
+- [Programming on Server-side](#programming-on-server-side)
+- [](#)
+- [](#)
+- [](#)
+
+### Introduction
+Socket programming is a way of connecting two computers on a network to allow the exchange of information between the two machines that are running over `JRE`(Java Runtime Environment). The connection can be through **connection-oriented protocol** or **connectionless** but for our case, we will be using connection-oriented which is TCP/IP.
```

Comment:
> to allow the exchange of information between the two machines that are running over `JRE`(Java Runtime Environment)
I would probably leave this part out because it sounds like you're saying that Socket programming is exclusive to the JRE and it can't be done in other environments. I would probably rephrase the sentence to something like: 
"Socket programming is a way of connecting two computers over a network to allow the exchange of information"